src/main/java/com/jadventure/game/conversation/ConversationManager.java

```diff
@@ -144,6 +128,32 @@ public void startConversation(NPC npc, Player player) throws DeathException {
         }
     }
 
+    public Line getLine(NPC npc, Player player, List<Line> conversation, Line start) {
+        for (Line l : conversation) {
+            if ("".equals(l.getPlayerPrompt()) &&
+                        ConversationManager.matchesConditions(npc, player, l)) {
+                start = l;
+                break;
+            }
+        }
+        return start;
+    }
```

Comment:
No need to pass `start` here
```suggestion
    public Line getLine(NPC npc, Player player, List<Line> conversation) {
        for (Line l : conversation) {
            if ("".equals(l.getPlayerPrompt()) &&
                        ConversationManager.matchesConditions(npc, player, l)) {
                return l;
            }
        }
        return null;
    }
```